### PR TITLE
Kehaar 050 upgrade

### PR DESCRIPTION
**Pull request by danielglauser**
[Pivotal Card](https://www.pivotaltracker.com/story/show/102524954)
Upgrades `election-works` to the latest Kehaar. Tested with `docker-compose` and in sie `krakenstein`. Project tested, Kraken approved.

### CODE REVIEW

File: src/election_works/channels.clj [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
   (:require [clojure.core.async :as async]))
 
-(def create-events (async/chan))
-(def update-events (async/chan))
-(def delete-events (async/chan))
+(defonce ok-requests (async/chan))
+(defonce ok-responses (async/chan))
+
+(defonce election-read-requests (async/chan))
+(defonce election-read-responses (async/chan))
```
Comment (tank157) [inline src/election_works/channels.clj]: There's some discussion going on over in https://github.com/democracyworks/election-authority-works/pull/7/files#r39280573 as to whether or not the responses channels need buffers.

File: src/election_works/queue.clj [hunk 2]
```diff
@@ -4,85 +4,97 @@
...
+      (partial handlers/delete-election channels/delete-events)))
+    (wire-up/start-responder!
+     channels/election-search-requests
+     channels/election-search-responses
+     (handlers/handle-errors handlers/search-election))
+    {:connections [connection]
+     :channels [(wire-up/incoming-service
+                 connection
+                 "election-works.ok"
+                 (config :rabbitmq :queues "election-works.ok")
```
Comment (tank157) [inline src/election_works/queue.clj]: Woop Woop, Scope Creep Off the Port Bow, Dive, Dive!

No, you didn't creep scope, I'm just going to suggest maybe we upgrade resource-config at this time too, in which case we'd need the vectorized parameters (and probably lots of other places). Feel free to fire a torpedo at this suggestion.

**Comment by danielglauser** [comment]
@tank157 This should be good to go sir.

**Comment by tank157** [comment]
I'm bringing this latest up in Krakenstein, it worked just fine prior to the last improvements around resource-config upgrade and adding response channel buffers, so I'm going to give this an LGTM for now, and come back if I find any problems running it.

**Comment by tank157** [comment]
I ran it again after the latest changes, didn't see any issues.

**Pull request merged**